chapter_13_database_layer_validation.asciidoc

```diff
@@ -521,6 +536,7 @@ class NewListTest(TestCase):
         self.assertEqual(Item.objects.count(), 0)
 ----
 ====
+// DAVID: Suggest using self.assertFalse(List.objects.exists()), etc.
```

Comment:
which assertion gives the most useful error message when it fails?  i'd have thought `assertEqual(...objects.all(), [])` actually....